README.md

```diff
@@ -296,14 +298,25 @@ Or the following `*.jsonschema.json` files, when the `strict` option is set to `
 
 The JSON Schema plugin supports the following options:
 
+- `bundle` - One of `true`, `false`, or `all`. Defaults to `all`. If `true`, only the bundle
+  schema will be generated. If `false`, only the non-bundle schema will be generated. If `all`, both
+  schemas will be generated. The bundle schema includes all dependencies in a single file, while the
+  non-bundle schema references the dependencies in separate files.
+- `names` - One of `json`, `proto`, or `all`, optionally suffixed with `-strict`. Defaults to `all`.
```

Comment:
Should this be a list (removing the `all` option)? Then the default could be `json,proto` but you could also mix and match if desired - i.e. `json,proto,proto-strict`.

Reply:
I need the 'all' option for the default, which will always output all combos
Note: You cannot specify proto-strict and proto together.

Reply:
If the option isn't specified you could default to all combos. I was thinking along the lines of https://github.com/bufbuild/protobuf-es/tree/main/packages/protoc-gen-es#import_extension which similarly allows configuring which output types should be generated.

Reply:
updated to match the 'target' argument you linked